jsamchineme/send-it
# Fix refresh token error

#### What does this PR do?
- Fix refresh token error
#### Description of Task to be completed?
- Remove previous token from the payload
#### How should this be manually tested?
- pull branch and run _npm run start:dev_
- make login request: POST http://localhost:8001/api/v1/auth/login request body: `{jsamchineme@gmail.com', password: 'secretpass'` and retrieve the token returned
- make token refresh request with the token returned: PATCH http://localhost:8001/api/v1/auth/refresh?token={returnedToken} and again retrieve the token returned
- repeat the last step at least 20 times and verify that the error is no longer produced
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162874164
#### Screenshots (if appropriate)
None
#### Questions:`
None
